src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -1091,6 +1092,15 @@ public static List<ClientTooltipComponent> gatherTooltipComponents(ItemStack sta
                 .toList();
     }
 
+    private static Stream<ClientTooltipComponent> splitLine(FormattedText text, Font font, int maxWidth)
+    {
+        if (text instanceof Component component && component.getContents() == ComponentContents.EMPTY && component.getSiblings().isEmpty())
```

Comment:
As another thought, could we take advantage of `Component#getString` as well for a separate condition and see whether the string is empty?